decidim-initiatives/decidim-initiatives.gemspec

```diff
@@ -29,8 +29,7 @@ Gem::Specification.new do |s|
   s.add_dependency "decidim-comments", Decidim::Initiatives.version
   s.add_dependency "decidim-core", Decidim::Initiatives.version
   s.add_dependency "decidim-verifications", Decidim::Initiatives.version
-  s.add_dependency "origami", "~> 2.1"
-  s.add_dependency "rexml", "~> 3.2.5" # Required for Origami gem to work with Ruby 3.0.0+
+  s.add_dependency "hexapdf", "~> 0.32.0"
```

Comment:
Let's remove the explicit dependency here as it's actually not being used in the code:
```suggestion
```

You need to run `bin/rake bundle`

Reply:
This is being used in decidim-initiatives/app/services/decidim/initiatives/pdf_signature_example.rb:22

Reply:
I agree with @alecslupu. Any library that is referred from the codebase should remain as a dependency.

If we want to completely get rid of this requirement, we should implement a contract for the signature service that the implementors could follow when adding it. By default we could ship a dummy service which does not do anything.

If we start removing dependencies that we are actually relying on, we will end up with this kind of problems:
https://github.com/decidim/decidim/pull/9511

This happens because during the code preloading process, these files can be required. And if you have installed `decidim-initiatives`, your production server would fail to start because it cannot find this dependency.

Reply:
Well, I would say that the scope of this PR is to swap origami with hexapdf. 
Regarding the testing part, i do not see any methods on how to successfully test file. 

On the "ship a dummy service which does not do anything." part, i would have to say is not in the scope of this PR, and it would be a decision that should be applied to as well to the following:
- `Decidim::Dev::DummyTranslator` - Actually does something, and it should not be a "Dummy"
- `Decidim::Initiatives::DummyTimestamp` - Actually does something 
- `Decidim::Initiatives::PdfSignatureExample` - Actually does something, and could be improved.

Reply:
Also, the scope of the `Decidim::Initiatives::PdfSignatureExample` is to showcase a functionality within the application. If we ship the dummy service, we loose the ability of showcasing that functionality.

Reply:
>  I would say that the scope of this PR is to swap origami with hexapdf.

I agree! Sorry for the noise.